src/plugins/sockets/inGameEvents.js

```diff
@@ -34,22 +35,18 @@ export async function handleInGameEvents(evData, newRoute = null) {
       gameStore.updateReady(evData.pNum);
       return;
     }
-    case SocketEvent.DEAL: {
-      // Update state
-      if (isSpectating) {
-        gameStore.myPNum = 0; // always spectate as p0
-        gameStore.isSpectating = true;
-      }
-      gameStore.resetPNumIfNullThenUpdateGame(evData.game);
-      break;
-    }
```

Comment:
Now that we trigger inGameEvents for potentially every state state, we should no longer separate deal as its own case. We should update every case to `resetPNumIfNullThenUpdateGame()`. Todo: clean these up to consolidate same-handled states and to ensure they all reset the pNum if null